docs/source/development/commands_reference.md

```diff
@@ -312,19 +322,20 @@ kedro run
-| `kedro run --runner runner_name`                                    | Run the pipeline with a specific runner                                                                                                                                                                                                                 | No                          |
-| `kedro run --env env_name`                                          | Run the pipeline in the env_name environment. Defaults to local if not provided                                                                                                                                                                         | No                          |
-| `kedro run --tag some_tag1,some_tag2`                               | Run only nodes which have any of these tags attached                                                                                                                                                                                                    | Yes                         |
-| `kedro run --load-version="some_dataset:YYYY-MM-DDThh.mm.ss.sssZ"`  | Specify a particular dataset version (timestamp) for loading                                                                                                                                                                                            | Yes                         |
-| `kedro run --pipeline de`                                           | Run the whole pipeline by its name                                                                                                                                                                                                                      | No                          |
-| `kedro run --config config.yml`                                     | Specify all command line options in a configuration file called config.yml                                                                                                                                                                              | No                          |
-| `kedro run --params param_key1:value1,param_key2:2.0`               | Does a parametrised kedro run with `{"param_key1": "value1", "param_key2": 2}`. These will take precedence over parameters defined in the `conf` directory. Additionally, dot (`.`) syntax can be used to address nested keys like `parent.child:value` | Yes                         |
+| CLI command                                                        | Description                                                                                                                                                                                                                                             | Multiple instances allowed? |
```

Comment:
I would be tempted to go a bit further with this table since we are editing it. I know you didn't write it @ankatiyar so you may not want to venture into the changes now, which is fine, but I wonder if it wouldn't make more sense to include a bit more standard layout.

So, rather than use examples (like `kedro run --pipeline de`) we should say (`kedro run --pipeline <pipeline_name>` and before the table starts, we make clear that anything within angle brackets is a placeholder for what the user would actually type.

I'm also confused about why there are 2 node names for a starting point (doesn't it start at a single place?) and some are ambiguous (`kedro run --config config.yml` -- can you call it something else or does it have to be `config.yml`). I wonder if we should add another column with examples or further descriptive text for each.

WDYT? Do you want to do this, or shall I put on the backlog for a future revision?

Reply:
Thanks @stichbury! I've updated the table. 

> I'm also confused about why there are 2 node names for a starting point (doesn't it start at a single place?) and some are ambiguous (`kedro run --config config.yml` -- can you call it something else or does it have to be `config.yml`). I wonder if we should add another column with examples or further descriptive text for each.

The workflow can start with multiple nodes - for example in the spaceflights tutorial `preprocess_companies_node` and `preprocess_shuttles_node` which are independent of each other. Running the project from one or the other will exclude other nodes on the same level and continue down to nodes on the next level. 
For `kedro run --config config.yml` the config file can have any name, I've updated the table to reflect this.